docs/DeveloperGuide.md

```diff
@@ -1,13 +1,39 @@
 # Developer Guide
 
+* Table of Contents
+{:toc}
```

Comment:
The table of contents (ToC) is unfilled. Perhaps it might be good to at least have a skeleton of the ToC at this point of time.